src/commands/lambda/constants.ts

```diff
@@ -135,18 +135,30 @@ export const SKIP_MASKING_ENV_VARS = new Set([
   VERSION_ENV_VAR,
 ])
 
+export enum Frameworks {
+  ServerlessFramework = 'Serverless Framework',
+  AwsCdk = 'AWS CDK',
+  AwsCloudFormation = 'AWS CloudFormation',
+  Unknown = 'Unknown',
+}
+
+// Mappings of files to frameworks.
+// For example, if `serverless.yml` exists, we know it's the Serverless Framework
+export const FRAMEWORK_MAPPINGS = new Map([
+  ['serverless.yaml', Frameworks.ServerlessFramework],
+  ['serverless.yml', Frameworks.ServerlessFramework],
+  ['serverless.js', Frameworks.ServerlessFramework],
+  ['cdk.json', Frameworks.AwsCdk],
+  ['.cdk.json', Frameworks.AwsCdk],
+  ['template.yaml', Frameworks.AwsCloudFormation],
+  ['template.yml', Frameworks.AwsCloudFormation],
+  ['template.json', Frameworks.AwsCloudFormation],
+])
```

Comment:
Why not reverse it?

export const FRAMEWORK_FILES_LOOKUP = {
  [DeploymentFrameworks.ServerlessFramework]: ['serverless.yaml', 'serverless.js', 'serverless.yml']
} as const

Reply:
For `O(1)` lookup, otherwise we have to loop through all the values to find out what the framework is.

Reply:
Sounds good, let's change the constant name to `FRAMEWORK_FILES_MAPPING`!